src/tls/keypair.h

```diff
@@ -63,6 +63,8 @@ namespace tls
     "-----BEGIN CERTIFICATE-----\n";
   static constexpr auto PEM_CERTIFICATE_FOOTER = "-----END CERTIFICATE-----\n";
 
+  using PbftSignature = std::array<uint8_t, MBEDTLS_ECDSA_MAX_LEN>;
```

Comment:
If we're defining `PbftSignatureSize`, we should use it here too (`= std::array<uint8_t, PbftSignatureSize>;`)

`PbftSignatureSize` is a value, not a type, so it should be `pbft_signature_size` (or maybe `PBFT_SIGNATURE_SIZE`, since it is macro-like).